### ISSUE TITLE
fail vagrant up provier

### ISSUE THREAD

**Issue by gearcapitan** [rapid7/metasploitable3#215]
## Issue Description
PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up
No usable default provider could be found for your system.

Vagrant relies on interactions with 3rd party systems, known as
"providers", to provide Vagrant with resources to run development
environments. Examples are VirtualBox, VMware, Hyper-V.

The easiest solution to this message is to install VirtualBox, which
is available for free on all major platforms.

If you believe you already have a provider available, make sure it
is properly installed and configured. You can see more details about
why a particular provider isn't working by forcing usage with
`vagrant up --provider=PROVIDER`, which should give you a more specific
error message for that particular provider.
PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up --provider=vmware
The provider 'vmware' could not be found, but was requested to
back the machine 'default'. Please use a provider that exists.
PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up --provider=VMware
The provider 'VMware' could not be found, but was requested to
back the machine 'default'. Please use a provider that exists.
PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up --provider=VMWARE
The provider 'VMWARE' could not be found, but was requested to
back the machine 'default'. Please use a provider that exists.
PS C:\Users\gearc\Documents\metasploitable3-master>
## Host System
* OS: windows 10
* Packer Version: last 
* Vagrant Version: last
* VirtualBox Version: last

## Command Output
Copy the relevant command output here.
If it's long, either post to a gist and add the link here, or isolate the error lines.

PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up
No usable default provider could be found for your system.

Vagrant relies on interactions with 3rd party systems, known as
"providers", to provide Vagrant with resources to run development
environments. Examples are VirtualBox, VMware, Hyper-V.

The easiest solution to this message is to install VirtualBox, which
is available for free on all major platforms.

If you believe you already have a provider available, make sure it
is properly installed and configured. You can see more details about
why a particular provider isn't working by forcing usage with
`vagrant up --provider=PROVIDER`, which should give you a more specific
error message for that particular provider.
PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up --provider=vmware
The provider 'vmware' could not be found, but was requested to
back the machine 'default'. Please use a provider that exists.
PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up --provider=VMware
The provider 'VMware' could not be found, but was requested to
back the machine 'default'. Please use a provider that exists.
PS C:\Users\gearc\Documents\metasploitable3-master> vagrant up --provider=VMWARE
The provider 'VMWARE' could not be found, but was requested to
back the machine 'default'. Please use a provider that exists.
PS C:\Users\gearc\Documents\metasploitable3-master>
![captura de pantalla 166](https://user-images.githubusercontent.com/13740942/34451198-aae5481c-ece4-11e7-97ed-dfc5d3976112.png)

**Comment by busterb** [comment]
Did you install vmware, or virtualbox, and are they accessible in your path? It's suggesting what the issue is.

Note, you can also just unzip the .box file that's created and import the VM directly.

**Comment by gearcapitan** [comment]
that's right, I had to manually decompress the virtual machine, but I have vmware and I only use vmware

**Comment by z3ndrag0n** [comment]
Just because you have vagrant and you have vmware doesn't mean you have the vagrant vmware provider - do you? (You'll know if you do, because it's not free.)

**Comment by Chan9390** [comment]
Hey @gearcapitan any updates ?

**Comment by gearcapitan** [comment]
@Chan9390  no :(

**Comment by nhegde610** [comment]
Are you using vagrant version 2.0.1 ?

**Comment by ttstormxx** [comment]
they did not make it clear that if you want vagrant to support vmware_desktop provider, you  need to pay for  the vagrant vmware desktop plugin.

**Comment by Chan9390** [comment]
Good point. Why not update the readme and create a PR ? If you have successfully setup Metasploitable 3 on VMware, you can also add that.
